docs/tools/cloud9.md

```diff
@@ -0,0 +1,38 @@
+# Cloud9 IDE
+
+Cloud9 is an open source, web based, IDE (integrated development environment) that allows for a developer to make changes
+to code without working from their desktop or needing additional tools.
+
+![Cloud9 IDE](../_img/cloud9-ide-screenshot.png)
+
+## Enabling Cloud9
+
+### Set the `IDE_ENABLED` variable for the project.
+
+Enable IDE for the project by setting the `IDE_ENABLED` variable.
+
+```bash
+fin config set --env=local IDE_ENABLED 1
+```
+
+!!! tip "For Other Developers on the Project"
+	To enable for other developers within the project remove the `--env=local` option from the above command.
```

Comment:
I think it's shorter and cleaner to write:

To enable for other developers within the project:

```bash
fin config set IDE_ENABLED=1
```